### ISSUE TITLE
Package to use/examine

### ISSUE THREAD

**Issue by lucasnell** [daijiang/phyr#11]
Joe just sent me a link to the [lme4ord package](https://stevencarlislewalker.github.io/lme4ord/) that might be of use. It runs GLMMs with "flexible covariance structures" and is based on lme4.

**Comment by arives** [comment]
Yeah, but I don't think it will be easy to get this to do nested matrices. Maybe Steve has broadened it, though.

**Comment by daijiang** [comment]
No update since 2018 though. I am not sure.

**Comment by lucasnell** [comment]
I'm not sure, but it says it includes phylogenetic covariance [here](https://stevencarlislewalker.github.io/lme4ord/analysts.html)

**Comment by daijiang** [comment]
This is the basic one. But it may be possible with the setReTrm generic function there. I am not sure.
